tests/python/sdk/test_tasks.py

```diff
@@ -292,6 +293,15 @@ def test_can_create_from_backup(self, fxt_new_task: Task, fxt_backup_file: Path)
         assert "100%" in pbar_out.getvalue().strip("\r").split("\r")[-1]
         assert self.stdout.getvalue() == ""
 
+    def test_can_create_from_backup(self, fxt_new_task: Task, fxt_backup_file: Path):
+        self._test_can_create_from_backup(fxt_new_task, fxt_backup_file)
+
+    def test_can_create_from_backup_in_chunks(
+        self, monkeypatch: pytest.MonkeyPatch, fxt_new_task: Task, fxt_backup_file: Path
+    ):
+        monkeypatch.setattr(Uploader, "_CHUNK_SIZE", 100)
+        self._test_can_create_from_backup(fxt_new_task, fxt_backup_file)
```

Comment:
Although the test is simple and, probably, correct, I think it will be reasonable to add a check that the expected tus requests were actually made. Could you please add an assertion about this? Otherwise, the test can silently start to check nothing.

Reply:
How could I assert that?

Reply:
My suggestion is to add a mock for the `Uploader._upload_file_data_with_tus()`, which just wraps it, and sets a test variable indicating the call was made. Other options - use pytest.mock.

Reply:
I went further and added a check that the upload was done in multiple chunks.

Reply:
Ok, looks good. Probably, can be improved somehow, but it should be enough for now.